back-end/helpers/user-helpers.js

```diff
@@ -18,6 +18,35 @@ async function getUserGitHubInfo(userID) {
   return userGitHubData;
 }
 
+async function getUserLanguage(userID) {
+  const query = 'SELECT language FROM github_info WHERE user_id = $1';
+  const userLanguage = await queryWithParameters(
+      query,
+      [userID]
+  );
+  return userLanguage.language;
```

Comment:
This variable `userLanguage` seems a little weird to me since we're trying to get the `language` property from `userLanguage`. Maybe it should be userProfile instead or something to that effect?

Reply:
Yeah I had originally just had it returning userLanguage then accessed the field later in the process so I can see how it looks weird now. Updated